### ISSUE TITLE
Workspaces button borders do not appear to be working

### ISSUE THREAD

**Issue by boderg** [Jas-SinghFSU/HyprPanel#416]
**Describe the bug**
When toggling the border button on the 'bar/layouts/enable button borders' all borders appear except for the workspaces border. The 'bar/workspaces/button border' toggle does not activate the border either.

**To Reproduce**
Toggle either of the buttons that turn on the associated border.

**Expected behavior**
Expected the border to toggle on and off.

**Screenshots**
![snapshot_2024-10-31_19-04-11](https://github.com/user-attachments/assets/5dfd23ba-4276-4d51-a7ac-57f1c1a15f25)

**Desktop (please complete the following information):**
 - Distribution: Arch Linux
 - Window Manager/Desktop Environment: Hyprland

**Additional context**

**Comment by Jas-SinghFSU** [comment]
Could you verify that:
1. You are on the latest version (`git pull`)
2. You have the right border color assigned in Theming > Bar > Workspaces > Border

**Comment by boderg** [comment]
I believe that I am on the latest pull.
Thank you for pointing me in the right direction and sorry if I have wasted any of your time. I have resolved this now and reset that one color.
Just to note this occurred after changing the theme to the one-dark theme, so I have looked at the theme and the color for the border is set to #21252b which is blending in with the background and my wallpaper. The reset has given me a contrasting color.
